app-forensics/scalpel/scalpel-2.1_pre20210326.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit autotools
+
+DESCRIPTION="A high performance file carver"
+HOMEPAGE="https://github.com/sleuthkit/scalpel"
+GITHASH="35e1367ef2232c0f4883c92ec2839273c821dd39"
+SRC_URI="https://github.com/sleuthkit/scalpel/archive/${GITHASH}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/scalpel-${GITHASH}"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="dev-libs/tre"
+RDEPEND="${DEPEND}"
+BDEPEND="${DEPEND}"
+
+PATCHES=( "${FILESDIR}/gcc-11-fix-literal-suffix.patch" )
```

Comment:
It appears that this patch could be replaced by a `sed`. This would save quite a few bytes in `FILESDIR`, since the patch is pretty large. Note that some prefer patches over sed invocations, but in this case, considering the patch size is around 10 KiB, and that it is a simple substitution for a fixed string ("`%"PRIu64 "`) with another fixed string ("`%" PRIu64 "`), I personally would tend to go with the sed.

Reply:
++